app/mailers/audit_mailer.rb

```diff
@@ -0,0 +1,21 @@
+class AuditMailer < ApplicationMailer
+  helper :audits, :layout
+
+  def summary(options = {})
+    raise ::Foreman::Exception.new(N_("Must specify a user with email enabled")) unless (user=User.find(options[:user])) && user.mail_enabled?
+    time     = options[:time] ? "time > \"#{options[:time]}\"" : 'time > yesterday'
+    @query    = "#{options[:query]} and #{time}"
+    @limit   = options[:limit] || 50
```

Comment:
This seems rather arbitrary, we should probably put a warning in the message if the limit has been reached, and the default should probably be `Setting[:entries_per_page]`